tutorials/physics/using_jolt_physics.rst

```diff
@@ -0,0 +1,272 @@
+default values.
+
+Single-body joints
+~~~~~~~~~~~~~~~~~~
+
+You can, in Godot, omit one of the joint bodies for a two-body joint and effectively
+have "the world" be the other body. However, the node path that you assign your body
+to (:ref:`node_a<class_Joint3D_property_node_a>` vs :ref:`node_b<class_Joint3D_property_node_b>`)
+is ignored. Godot Physics will always behave as if you
+assigned it to node_a, and since node_a is also what defines the frame of reference
+for the joint limits, you end up with inverted limits and a potentially strange
+limit shape, especially if your limits allow both linear and angular degrees of
+freedom.
+
+Jolt will behave as if you assigned the body to node_b instead, with node_a
+representing "the world". There is a project setting called :ref:`Physics > Jolt Physics 3D > Joints > World Node<class_ProjectSettings_property_physics/jolt_physics_3d/joints/world_node>`
+that lets you toggle this behavior, if you need compatibility for an existing project.
+
+Collision margins
+~~~~~~~~~~~~~~~~~
+
+Jolt (and other similar physics engines) uses something that Jolt refers to as
+"convex radius" to help improve the performance and behavior of the types of
+collision detection that Jolt relies on for convex shapes. Other physics engines
+(Godot included) might refer to these as "collision margins" instead. Godot exposes
+these as the ``margin`` property on every Shape3D-derived class, but Godot Physics
+itself does not use them for anything.
+
+What these collision margins sometimes do in other engines (as described in Godot's
+documentation) is effectively add a "shell" around the shape, slightly increasing
+its size while also rounding off any edges/corners. In Jolt however, these margins
+are first used to shrink the shape, and then the "shell" is applied, resulting in
+edges/corners being similarly rounded off, but without increasing the size of the
+shape.
+
+To prevent having to tweak this margin property manually, since its default value
+can be problematic for smaller shapes, this module exposes a project setting called
+:ref:`Physics > Jolt Physics 3D > Collisions > Collision Margin Fraction<class_ProjectSettings_property_physics/jolt_physics_3d/collisions/collision_margin_fraction>`
+which is multiplied with the smallest axis of the shape's AABB to calculate the
+actual margin. The margin property of the shape is then instead used as an upper
+bound.
+
+These margins should, for most use-cases, be more or less transparent, but can
+sometimes result in odd collision normals when performing shape queries. You can
+lower the above mentioned project setting to mitigate some of this, including
+setting it to ``0.0``, but too small of a margin can also cause odd collision results,
+so is generally not recommended.
+
+Baumgarte stabilization
+~~~~~~~~~~~~~~~~~~~~~~~
+
+the Baumgarte stabilization Jolt uses can result in some artifacts, like piles of
+bodies not separating as quickly as one might expect.
```

Comment:
```suggestion
Baumgarte stabilization is a method to resolve penetrating bodies and push them in a state where they are just touching. In Godot Physics this works like a spring. This means that bodies can accellerate and may cause the bodies to overshoot and separate completely. With Jolt, the stabilization is only applied to the position and not to the velocity of the body. This means it cannot overshoot but it may take longer to resolve the penetration.
```

Reply:
I can't speak to the technical details, but it seems alright to me.

I still think we need to rethink this section in general, as people who find the slower convergence to be a problem aren't likely to look for Baumgarte stabilization in the documentation, but that can be done later.

Minor spelling/grammatical notes:
- "push them in a state" -> "push them to a state"?
- "accellerate" -> "accelerate"